google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -803,7 +803,11 @@ private SpanName getSpanName(String methodName) {
   @Override
   public void close() {
     for (BackgroundResource backgroundResource : clientContext.getBackgroundResources()) {
-      backgroundResource.shutdown();
+      try {
+        backgroundResource.close();
```

Comment:
Just wondering, should we also update admin clients to call close instead of shutdown also?  (for example: https://github.com/googleapis/java-bigtable/blob/master/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/GrpcBigtableInstanceAdminStub.java#L831)

Reply:
Yep, but that needs to be done in the gapic-generator:
https://github.com/googleapis/gapic-generator-java/issues/792

But its less of a priority